templates/tools/dockerfile/test/cxx_jessie_x64/Dockerfile.template

```diff
@@ -25,7 +25,7 @@
   <%include file="../../libuv_install.include"/>
 
   # Install gcc-4.8 and other relevant items
-  RUN apt-get update && apt-get -y install gcc-4.8 gcc-4.8-multilib g++-4.8 g++-4.8-multilib && apt-get clean
+  RUN apt-get update && apt-get -y install gcc-4.8 gcc-4.8-multilib g++-4.8 g++-4.8-multilib iptables && apt-get clean
```

Comment:
let's add iptables on a separate line and add an explanatory comment.

`RUN apt-get update && apt-get -y install iptables && apt-get clean`